README.md

```diff
@@ -414,10 +414,12 @@ were necessary.
 Each commit should ideally contain changes that are functionally
 connected/related.  For example, changes to the command line parsing
 code, changes to the documentation, and changes to some unrelated
-comments may be split into three commits.
+comments may be split into three commits. Also note that a single logical
+change (suitable for a single commit) may comprise changes in several
+files.
 
 Learn how to select chunks of changed files to do multiple
-separate commits of unrelated things. This is done using `git commit -p
+separate commits of unrelated things. This can be done using `git add -p
```

Comment:
What's the benefit of using `add` in place of `commit`?

Reply:
Yep, maybe that's better kept in a separate paragraph and more explicit (I consider it an improvement that `git add` adds to the staged changes whereas `git commit` can't be partial, so if one aborts, the would-be-commited changes are not kept).

Reply:
I'd like to see `git commit -p` still mentioned. I've tried `git add -p`, but as this does not appear to actually commit anything, it seem to be only half of a `commit` operation.

Reply:
I've added some expansion about git add and how you can not use it.